internal/web3ext/web3ext.go

```diff
@@ -932,6 +932,16 @@ web3._extend({
 			params: 1,
 			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
 		}),
+		new web3._extend.Method({
+			name: 'setRollupEventSyncedL1Height',
```

Comment:
How would one use these 2 new APIs?

The problem is that both modules only read from db during startup and then use the value stored in memory.

So updating DB will not automatically let the node resync the data, it would need to be restarted first. But between the RPC call and node restart, the value might be overwritten.

Would this be a problem in practice?

Reply:
nice catch. maybe not using rpc, but using geth db cmd instead. will investigate other methods later.

Reply:
the implementation is changed in [d38f656](https://github.com/scroll-tech/go-ethereum/pull/1044/commits/d38f65619b441b073564ede3ae039e2b07b2010e). in error status, usually, the sync service will be stuck at some error paths, thus: (i) full stop; (ii) reset db; (iii) restart service; should be fine.